tox.ini

```diff
@@ -193,7 +193,7 @@ pass_env=
   PHOENIX_SMTP_USERNAME
   PHOENIX_SMTP_PASSWORD
 commands_pre =
-  uv tool install arize-phoenix@. \
+  uv tool install -U --force arize-phoenix@. \
```

Comment:
Doesn't this kill the cache?

Reply:
Our deps don't upgrade otherwise. Was causing crashes when running with pnpm dev 

It seemed to still cache for me until a dep needed upgrading but someone with a cache will need to pull down and see what pnpm dev does

Reply:
Okay cool - I just hate it when I can't run phoenix on planes. haha